<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="Draw" name="Canvas" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_generate_points">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Generate points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_select_method_qp">
       <property name="text">
        <string>Select method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_generateMethod">
       <item>
        <property name="text">
         <string>raster</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>random</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>circle</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_select_count">
       <property name="text">
        <string>Select count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_countPoints">
       <item>
        <property name="text">
         <string>1000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>50000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>75000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>100000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>250000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>500000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>750000</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1000000</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_generatePoints">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Generate points</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_create_convex_hull">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create convex hull</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_select_method_ch">
       <property name="text">
        <string>Select method:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_chMethod">
       <item>
        <property name="text">
         <string>Jarvis Scan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quick Hull</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sweep Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Graham Scan</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_solveU2">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Solve U2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_createCH">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Create convex hull</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_text_elapsed_time">
       <property name="text">
        <string>Time elapsed(ms):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_timeElapsed">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clearCH">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Clear convex hull</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_clearPoints">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Clear points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Draw</class>
   <extends>QWidget</extends>
   <header location="global">draw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
